tools/pubsys/src/aws/ssm/ssm.rs

```diff
@@ -97,7 +94,7 @@ where
         ensure!(
             total_count == expected_len,
             error::MissingInResponseSnafu {
-                region: region.name(),
+                region: &region,
```

Comment:
nit: can we do this?:

```suggestion
                region,
```